modules/Vtiger/uitypes/String.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * String UIType field file.
+ *
+ * @package   UIType
+ *
+ * @copyright YetiForce S.A.
+ * @license   YetiForce Public License 5.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Adrian Kon <a.kon@yetiforce.com>
+ */
+
+/**
+ * String UIType field class.
+ */
+class Vtiger_String_UIType extends Vtiger_Base_UIType
+{
+	/** {@inheritdoc} */
+	protected $isResizableColumn = true;
+
+	/** {@inheritdoc} */
+	public function validateMaximumLength(int $minimumLength, int $maximumLength): bool
+	{
+		$allowedMaxValue = 255;
+		$newColumnLength = (int) $maximumLength;
+		return $newColumnLength <= $allowedMaxValue;
+	}
+
+	/** {@inheritdoc} */
+	public function changeMaximumLength(int $minimumLength, int $maximumLength): void
+	{
+		if ($this->isResizableColumn() && $this->validateMaximumLength($minimumLength, $maximumLength)) {
+			$fieldInstance = $this->getFieldModel();
+			if ($this->isColumnLengthIncreased($maximumLength)) {
```

Comment:
Do you really need the "isColumnLengthIncreased" function?
you call the getDBColumnType(false) function anyway.